homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -193,6 +201,7 @@ def supported_features(self) -> int:
             SUPPORT_ALARM_ARM_HOME
             | SUPPORT_ALARM_ARM_AWAY
             | SUPPORT_ALARM_ARM_NIGHT
+            | SUPPORT_ALARM_ARM_VACATION
```

Comment:
In a follow-up PR we should make the MQTT alarm control panel only report features that it actually supports.